dev/com.ibm.ws.security.fat.common/src/com/ibm/ws/security/fat/common/TestServer.java

```diff
@@ -754,11 +757,42 @@ public void stopServer() throws Exception {
             // ignore ssl message - runtime retries and can proceed (sometimes) when it can't tests will fail when they don't get the correct response
             server.addIgnoredErrors(Arrays.asList(MessageConstants.CWWKO0801E_UNABLE_TO_INIT_SSL));
 
+            shibbolehtBackup(server);
+
             server.stopServer(ignoredServerExceptions);
         }
         unInstallCallbackHandler(callback, callbackFeature);
     }
 
+    /**
+     * backup shibboleth config
+     */
+    private void shibbolehtBackup(LibertyServer server) throws Exception {
```

Comment:
FYI. Obviously not a functional worry.
```suggestion
    private void shibbolethBackup(LibertyServer server) throws Exception {
```